wagtail/admin/static_src/wagtailadmin/scss/userbar.scss

```diff
@@ -137,7 +137,7 @@ $positions: (
         left: 0;
     }
 
-    &.#{$namespace}-icon:before {
+    &. #{$namespace}-icon:before {
```

Comment:
Typo? The dot needs to be at the start of the class name, and `&` is useless if not directly joined with the rest of the selector.